spec/features/project_spec.rb

```diff
@@ -0,0 +1,98 @@
+require "rails_helper"
+
+RSpec.feature "Projects", type: :feature do
+  it "can see a project's name and material" do
+    recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+    furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+    project_1 = recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+    material = project_1.material
+    
+    visit "/project/#{project_1.id}"
+
+    expect(page).to have_content(project_1.name)
+    expect(page).to have_content(project_1.material)
+  end
+
+#   User Story 3 of 3
+# As a visitor,
+# When I visit a project's show page
+# I see a count of the number of contestants on this project
+
+# (e.g.    Litfit
+#     Material: Lamp Shade
+#   Challenge Theme: Apartment Furnishings
+#   Number of Contestants: 3 )
+
+  it "can see a count of the number of contestants on this project" do
+    recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
```

Comment:
I know you're under a time crunch with an IC, but in this type of test in general it's really helpful to have a `before(:each)` so that you don't have to repeat the test setup in every test case